src/core/components/hv-doc/index.tsx

```diff
@@ -226,27 +269,24 @@ const HvDoc = (props: Props) => {
       getNavigation,
       getOnUpdate: () => onUpdate,
       getState: getScreenState,
-      setNeedsLoad: needsLoadCallback.current,
       setState: setScreenState,
+      updateUrl: (url: string) => {
+        setLoadingUrl(url);
+      },
```

Comment:
minor: can we pass a reference of the function instead of creating an anonymous wrapper?
```suggestion
      updateUrl: setLoadingUrl,
      },
```

Reply:
Will take a look

Reply:
https://github.com/Instawork/hyperview/pull/1158/commits/55aa48a77c984e0f144c0ba32f08ba61e0ea6733